src/Game.js

```diff
@@ -0,0 +1,90 @@
+function Game() {
+
+  this.score = 0;
+  this.frameScore = 0;
+  this.frameNumber = 1;
+  this.firstThrow = true;
+  this.spareBall = false;
+  this.strikeBall = false;
+  this.tenthFrameBonus = false;
+  this.gameOver = false;
+  this.scoreBoard = {1:[], 2:[], 3:[], 4:[], 5:[], 6:[], 7:[], 8:[], 9:[], 10:[]}
+};
+
+Game.prototype.rollBall =  function(hit) {
+
+  if ((this.frameNumber < 10) && (this.firstThrow === false) && (this.frameScore + hit > 10)) {
+    throw Error("Not enough pins left");
+
+  } else if (this.frameNumber < 10) {
+      this.addToScoreBoard(hit);
+      this.frameScore += hit;
+      this.isSpare(hit);
+      this.isFirstThrow();
+      this.isStrike(hit);
+      this.resetFrameScore();
+
+  } else if (this.frameNumber === 10 && this.gameOver === false) {
+    this.addToScoreBoard(hit);
+    this.frameScore += hit;
+    this.isSpare();
+    this.isStrike();
+    this.game();
+  };
+};
+
+Game.prototype.addToScoreBoard = function(hit) {
+  if (this.strikeBall === true) {
+    this.scoreBoard[this.frameNumber].push(hit*2);
+    this.score += (hit*2);
+    this.strikeBall = false;
+  } else if (this.spareBall === true) {
+    this.scoreBoard[this.frameNumber].push(hit*2);
+    this.score += (hit*2);
+    this.spareBall = false;
+  } else {
+    this.scoreBoard[this.frameNumber].push(hit);
+    this.score += (hit);
+  };
+};
+
+Game.prototype.isFirstThrow = function() {
```

Comment:
It's not a very good idea to name things that change state with `is` because that suggests that you are checking something